CMakeLists.txt

```diff
@@ -109,8 +129,22 @@ endif()
    INCLUDE (${g3log_SOURCE_DIR}/test_unit/Test.cmake)
 
 
+if( WIN32 AND G3LOG_SHARED_LIBRARY )
+   install( PROGRAMS $<TARGET_PDB_FILE:g3logger_shared>
+		DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+		COMPONENT libraries
+	)
+endif()
 
+INSTALL( TARGETS g3logger ${G3LOG_SHARED_LIBRARY}
```

Comment:
please explain the rationale between moving the install instructions from the cpack to the cmake file